modules/apps/packet-forward-middleware/keeper/keeper.go

```diff
@@ -0,0 +1,469 @@
+package keeper
```

Comment:
One thing I am realizing, is that the keeper should probably only use `InternalTransferRepresentation` to make it easier to migrate to IBC v2 later (and to more easily reuse transfer logic we have for different things)

This could be a separate task though.

wdyt @AdityaSripal?